bin/crew

```diff
@@ -909,6 +909,17 @@ def prepare_package(destdir)
     system 'find . -type l >> ../filelist'
     system 'cut -c2- ../filelist > filelist'
 
+    # check for FHS3 compliance
+    @fhs_compliant_prefix = ['bin', 'etc', 'include', 'lib', 'libexec', 'opt', 'sbin', 'share', 'var']
+    @fhs_compliant_prefix.append(ARCH_LIB) unless ARCH_LIB == 'lib'
```

Comment:
It would be clearer to use `if ARCH_LIB == 'lib64'`

Reply:
baafd1f